matisse/src/main/java/com/zhihu/matisse/internal/model/AlbumCollection.java

```diff
@@ -23,25 +23,36 @@
 import android.support.v4.app.LoaderManager;
 import android.support.v4.content.Loader;
 
+import com.zhihu.matisse.internal.entity.Item;
 import com.zhihu.matisse.internal.loader.AlbumLoader;
+import com.zhihu.matisse.internal.ui.MediaSelectionFragment;
 
 import java.lang.ref.WeakReference;
+import java.util.ArrayList;
 
 public class AlbumCollection implements LoaderManager.LoaderCallbacks<Cursor> {
     private static final int LOADER_ID = 1;
     private static final String STATE_CURRENT_SELECTION = "state_current_selection";
     private WeakReference<Context> mContext;
```

Comment:
A little question not related to this PR, I've noticed that `WeakReference` is widely used in this project, just like the mentioned line. I guess you maybe want to break the cycle reference between them, but in my opinion, this is unnecessary, because GC could clean all objects not (directly or indirectly) linked with **GC Root**, `Activity` instances (after finished) and instances of this class are not GC Root, so they can be cleaned by GC normally.